build/Dockerfile

```diff
@@ -19,29 +14,19 @@ COPY src src/
 COPY pom.xml ./
 COPY license-checks.xml ./
 COPY java.header ./
-
-# Install google-cloud-sdk to get gcloud.
-RUN mkdir -p /usr/local/gcloud                                              && \
-    cd /usr/local/gcloud                                                    && \
-    curl -s -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-    tar -xf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz       && \
-    /usr/local/gcloud/google-cloud-sdk/install.sh                           && \
-    ln -s /usr/local/gcloud/google-cloud-sdk/bin/gcloud /usr/bin/gcloud     && \
-    rm google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz
+# Download dependencies
+RUN mvn dependency:go-offline
 
 # Build from source.
-RUN mvn clean package -P shade -DskipTests
+RUN mvn package -Passembly -DskipTests
 
 ################################################################################
 #                                   RELEASE                                    #
 ################################################################################
 
-FROM gcr.io/cloud-marketplace/google/ubuntu1804:latest
-
-RUN apt-get update                                                          && \
-    apt-get -qq -y install default-jre
+FROM eclipse-temurin:17-jre
```

Comment:
curious, do we need eclipse for JRE ?

Reply:
`eclipse` is just the name of the Docker image, it does not contain the `eclipse` IDE. It is maintained by the Eclipse Software Foundation: https://adoptium.net/docs/faq